deploy.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+UPDATE=$1
+DEST_DIR=$2
+FED_KEY=""
+MAIN_BLOCK_NUMBER=""
+SIDE_BLOCK_NUMBER=""
+
+cd $DEST_DIR/federator
+
+quit() {
+    if [ $1 -eq 1 ]; then
+        echo "ERROR: "$2
+    else
+        echo $2
+    fi
+    exit $1
+}
+
+config() {
+    cp $DEST_DIR/federator/config/config.sample.js $DEST_DIR/federator/config/config.js &&
+        sed -i "s|rsktestnet-kovan|rskmainnet|g" $DEST_DIR/federator/config/config.js &&
+        sed -i "s|kovan|ethmainnet|g" $DEST_DIR/federator/config/config.js &&
```

Comment:
This could be replaced to something like

```
sed -e 'first replacement' -e 'seconde replacement' $source_file >$dest_file
```

but am fine either way.